src/Symfony/Bundle/SecurityBundle/composer.json

```diff
@@ -18,7 +18,8 @@
     "require": {
         "php": ">=5.3.9",
         "symfony/security": "~2.8|~3.0.0",
-        "symfony/http-kernel": "~2.2|~3.0.0"
+        "symfony/http-kernel": "~2.2|~3.0.0",
+        "symfony/security-acl": "~2.8|~3.0.0"
```

Comment:
To preserve backwards compatibility as it's always available atm. Once the Acl Bundle is created, I can move this dependency there and put the bundle as dependency in the SecurityBundle. In 3.0 the BC layer (regarding config) + dependency can be removed.